products/mongodb.md

```diff
@@ -36,8 +36,8 @@ releases:
     eol: false
 -   releaseCycle: "6.0"
     eol: 2025-07-01
-    latest: "6.0.3"
-    latestReleaseDate: 2022-11-14
```

Comment:
MongoDB description could mention the Enterprise Server edition (and maybe use the extended support column).

Reply:
#2320